src/firefox/remote.js

```diff
@@ -1,6 +1,9 @@
 /* @flow */
 import {createLogger} from '../util/logger';
-import {RemoteTempInstallNotSupported, WebExtError} from '../errors';
+import {RemoteTempInstallNotSupported,
+        UsageError,
+        WebExtError,
+        } from '../errors';
```

Comment:
To remain consistent with the rest of the code, this should be formatted like:
````
import {
  RemoteTempInstallNotSupported,
  UsageError,
  WebExtError,
} from '../errors';
````